### PR TITLE
Update integration-management with note about partner ints

### PR DESCRIPTION
**Pull request by eho1307**
### What does this PR do? What is the motivation?

Add a clarifying note that partner/community integrations need to be re-installed after the agent has been upgraded

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/agent/guide/integration-management.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ algolia:
 
 The Agent comes with a set of bundled official Datadog integrations to allow users to start monitoring their applications quickly. These integrations are available as single Python packages, and you can upgrade them separately.
 
+**Note:** Non-core integrations such as those built by partners or community members are not retained when the Agent is upgraded. Please be mindful that these integrations need to be re-installed upon upgrading the Agent version.
```
Comment (cswatt) [inline content/en/agent/guide/integration-management.md:12]:
```suggestion
**Note**: Community and Marketplace integrations are not retained when the Agent is upgraded. These integrations need to be re-installed upon upgrading the Agent version.
```

File: content/en/agent/troubleshooting/integrations.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ Datadog integrations are configured through the Datadog Agent using YAML configu
 
 If an integration that you've configured is not showing up in Datadog, run the [`status` CLI command][2] and look for the integration under the *Running Checks* heading.
 
+**Note:** Non-core integrations such as those built by partners or community members are not retained when the Agent is upgraded. Please be mindful that these integrations need to be re-installed upon upgrading the Agent version.
```
Comment (cswatt) [inline content/en/agent/troubleshooting/integrations.md:22]:
```suggestion
**Note**: Community and Marketplace integrations are not retained when the Agent is upgraded. These integrations need to be re-installed upon upgrading the Agent version.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
I think we should use "Community and Marketplace integrations" rather than "non-core integrations"—the word "core" might have meaning to us internally since the repo is called `integrations-core`, but users may not understand what it means.

Also removing use of "please" as per docs style guide.

File: content/en/agent/guide/integration-management.md (hunk 1 above)
Comment (eho1307) [reply content/en/agent/guide/integration-management.md:12]: @cswatt what do you think of **Note**: Community, Partner and Marketplace integrations are not retained when the Agent is upgraded. These integrations need to be re-installed upon upgrading the Agent version.

**Review by cswatt — APPROVED** [review]

**Pull request merged**